products/BellHybrid/apps/common/include/common/data/BellFinishedWindowSwitchData.hpp

```diff
@@ -13,24 +13,31 @@ namespace gui
     struct BellFinishedWindowData : public gui::SwitchData
     {
       public:
+        enum class ExitBehaviour
+        {
+            SwitchWindow,
+            CloseApplication,
+            ReturnToHomescreen,
+        };
+
         struct Factory
         {
             static std::unique_ptr<BellFinishedWindowData> create(
```

Comment:
to be pedantic I would make 2 create functions( one taking 2 parameters, second taking 5 parameters) to avoid default values, but its stylistic thing

Reply:
Yep, it's a good idea.

Reply:
@swichu91 @disonek 
please check changes. I made separate commit for it to be more visible